tests/handlers/test_receipts.py

```diff
@@ -20,6 +20,10 @@
 
 from tests import unittest
 
+# In the first param of _test_filters_hidden we use "hidden" instead of
+# ReadReceiptEventFields.MSC2285_HIDDEN. We do this because we're mocking
+# the data from the database which doesn't use the prefix
+
```

Comment:
I'm going to be annoying again (sorry :sweat_smile:) but can you just put it between `prepare` and `test_filters_out_hidden_receipt`? The intention is that it's as fresh as possible in the mind of people when they start reading the actual tests. Sorry I wasn't clear enough!

Reply:
Absolutely no worries! :smile: